openssl-dynamic/src/main/c/ssl.c

```diff
@@ -54,6 +56,17 @@ struct CRYPTO_dynlock_value {
     apr_thread_mutex_t *mutex;
 };
 
+struct TCN_bio_bytebuffer {
```

Comment:
@Scottmitch I wonder if you maybe want to move the while own bio type thing into another file ?